pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -117,6 +118,25 @@ func getContainerLogsPath(containerName string, podUID types.UID) string {
 	return path.Join(podLogsRootDirectory, string(podUID), fmt.Sprintf("%s.log", containerName))
 }
 
+// applyContainerConfigAnnotations applies some container configurations into annotations. These configurations
+// are usually experimental features. They'll be promoted to official runtime api in the future.
+func applyContainerConfigAnnotations(annotations map[string]string, pod *api.Pod, container *api.Container) {
```

Comment:
It seems that we are not passing `pod.Annotations` to container, so we may still need this function.